src/payload/orderPayload.ts

```diff
@@ -0,0 +1,41 @@
+import OrderPayloadBuilder from "./orderPayloadBuilder";
+import { Sender } from "../models/sender";
+import { Recipient } from "../models/recipient";
+
+export default class OrderPayload {
+    private quotationId: string | undefined;
+
+    private sender: Sender | undefined;
+
+    private recipients: Recipient[] | undefined;
+
+    private fleetOption?: string;
+
+    private isPODEnabled?: boolean;
+
+    private isRecipientSmsEnabled?: boolean;
+
+    private paymentMethod?: string;
+
+    private partner?: string;
+
+    constructor(opb: OrderPayloadBuilder) {
+        if (opb.quotationId === undefined) {
```

Comment:
Very small detail but let's use the `Yoda` style conditions
https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
It is not hard to change it but why do you think we should use it?